platforms/hyperledger-fabric/charts/invoke_chaincode/templates/invoke_chaincode.yaml

```diff
@@ -128,19 +165,45 @@ spec:
         command: ["sh", "-c"]
         args:
         - |-
-          #!/bin/bash
+          #!/bin/bash sh
+          set -x
           export ARGS='{"Args":'"'["${INSTANTIATION_ARGUMENTS}"]'"'}'
           export qARGS="'${ARGS}'"
-          touch ./invokeChaincode.sh
+
+          touch ./invokeChaincode.sh	
+          
           version1_4=`echo $NETWORK_VERSION | grep -c 1.4`
           if [ $version1_4 = 1 ];then
             echo "peer chaincode invoke -o ${ORDERER_URL} --tls ${CORE_PEER_TLS_ENABLED} --cafile ${ORDERER_CA} -C ${CHANNEL_NAME} -n ${CHAINCODE_NAME} -c $qARGS" >> ./invokeChaincode.sh
+            chmod 755 ./invokeChaincode.sh
+            sh ./invokeChaincode.sh
           else
-            echo "peer chaincode invoke -o ${ORDERER_URL} --tls ${CORE_PEER_TLS_ENABLED} --cafile ${ORDERER_CA} -C ${CHANNEL_NAME} -n ${CHAINCODE_NAME} --isInit -c $qARGS" >> ./invokeChaincode.sh
+            INVOKE_CMD="peer chaincode invoke -o ${ORDERER_URL} --tls ${CORE_PEER_TLS_ENABLED} --cafile $ORDERER_CA --channelID ${CHANNEL_NAME} --name ${CHAINCODE_NAME} --isInit -c $qARGS"
+            
+            for item in $ENDORSERS_CORE_PEER_ADDR
+            do
+              INVOKE_CMD=${INVOKE_CMD}" --peerAddresses "${item}
+            done
+
+            echo ${INVOKE_CMD}
```

Comment:
Do the same here, some of the logs are for debugging purposes, either remove or make it more descriptive